src/bitcoin-tx.cpp

```diff
@@ -158,7 +159,7 @@ static void RegisterLoad(const std::string& strInput)
     std::string key = strInput.substr(0, pos);
     std::string filename = strInput.substr(pos + 1, std::string::npos);
 
-    FILE *f = fopen(filename.c_str(), "r");
+    FILE *f = fsbridge::fopen(filename.c_str(), "r");
```

Comment:
unrelated question for follow-up: Why is this safe? Shouldn't this use PathFromString?